### PR TITLE
Buttons!

### PR DESCRIPTION
**Pull request by designatednerd**
Adds buttons which open the URL scheme with fake info to the sample applications.

### CODE REVIEW

**Comment by designatednerd** [comment]
@vokal/ios-developers @vokal/android-developers Who wants to look at a bunch of XML?!

**Comment by brockboland** [comment]
Looks buttony to me!

File: iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="6254" systemVersion="14C109" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="vXZ-lx-hvc">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="6751" systemVersion="14C1510" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="vXZ-lx-hvc">
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="6247"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="6736"/>
+        <capability name="Constraints to layout margins" minToolsVersion="6.0"/>
```
Comment (chillpop) [inline iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard]: Your min target is iOS7, right? (according to the podspec at least)
If so, the margins need to go.

File: iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -69,19 +44,59 @@
...
+                            <constraint firstItem="6qO-ak-zP6" firstAttribute="centerX" secondItem="8uc-id-dgY" secondAttribute="centerX" id="QtO-kB-xDO"/>
+                            <constraint firstItem="6qO-ak-zP6" firstAttribute="top" secondItem="jyV-Pf-zRb" secondAttribute="bottom" constant="26" id="T24-24-PNP"/>
+                            <constraint firstItem="UWH-c9-Arn" firstAttribute="top" secondItem="vCw-Sm-PPb" secondAttribute="bottom" constant="16" id="ZnV-7E-08a"/>
+                            <constraint firstItem="UWH-c9-Arn" firstAttribute="centerX" secondItem="kh9-bI-dsS" secondAttribute="centerX" id="eMU-FS-d0b"/>
+                            <constraint firstItem="sl2-rz-Ntj" firstAttribute="centerX" secondItem="8uc-id-dgY" secondAttribute="centerX" id="kQw-qH-uch"/>
+                            <constraint firstItem="vCw-Sm-PPb" firstAttribute="top" secondItem="sl2-rz-Ntj" secondAttribute="bottom" constant="17" id="kaY-y3-zkQ"/>
+                            <constraint firstItem="8uc-id-dgY" firstAttribute="leading" secondItem="kh9-bI-dsS" secondAttribute="leadingMargin" constant="4" id="p2C-M4-58z"/>
```
Comment (chillpop) [inline iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard]: the offending constraint

**Comment by DavidRyan** [comment]
androidy part lgtm

File: iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard (hunk 1 above)
Comment (designatednerd) [inline iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard]: This is just a sample project, so it's targeting 8. The pod only contains non-UI files.

File: iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard (hunk 2 above)
Comment (designatednerd) [inline iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard]: (same deal)

File: iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard (hunk 1 above)
Comment (chillpop) [inline iOS/PartnerURLSchemeSample/PartnerURLSchemeSample/Base.lproj/Main.storyboard]: in that case LGTM

**Comment by designatednerd** [comment]
Gracias amigos.

**Pull request merged**